DESCRIPTION

```diff
@@ -39,3 +39,4 @@ Roxygen: list(markdown = TRUE)
 License: GPL-3
 Encoding: UTF-8
 RoxygenNote: 7.3.1
+Additional_repositories: https://p3m.dev/cran/2024-02-02/
```

Comment:
CRAN acknowledges this field but doesn't use it. `install.packages()` does not use it. `remotes::install_*()` *does* use it, but `pak::pak()` does not (yet) :shrug:

This PPPM repo is pinned to the current arrow release version. Alternatively, you can instead get the latest pre-release version of arrow by instead using `Additional_repositories: https://apache.r-universe.dev/` or `Additional_repositories: https://nightlies.apache.org/arrow/r`.